integration/ingester_test.go

```diff
@@ -0,0 +1,384 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+// Provenance-includes-location: https://github.com/cortexproject/cortex/blob/master/integration/ingester_sharding_test.go
+// Provenance-includes-license: Apache-2.0
+// Provenance-includes-copyright: The Cortex Authors.
+//go:build requires_docker
+
+package integration
+
+import (
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/grafana/e2e"
+	e2edb "github.com/grafana/e2e/db"
+	"github.com/prometheus/common/model"
+	"github.com/prometheus/prometheus/model/labels"
+	"github.com/prometheus/prometheus/prompb"
+	"github.com/prometheus/prometheus/storage/remote"
+	"github.com/stretchr/testify/require"
+
+	"github.com/grafana/mimir/pkg/util/test"
+
+	"github.com/grafana/mimir/integration/e2emimir"
```

Comment:
not sure why this wasn't linted

```suggestion
	"github.com/grafana/mimir/integration/e2emimir"
	"github.com/grafana/mimir/pkg/util/test"
```